mods/d2k/rules/aircraft.yaml

```diff
@@ -1,4 +1,4 @@
-carryall.scripted:
+carryall.reinforce:
```

Comment:
This should probably not have `TargetableAircraft`. `carryall.infantry` neither, come to think of it.